### ISSUE TITLE
Uncaught TypeError: Object.defineProperty called on non-object

### ISSUE THREAD

**Issue by PowerlessCube** [AtomLinter/linter-jscs#274]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.8.0
**System**: Mac OS X 10.11.5
**Thrown From**: [linter-jscs](https://github.com/AtomLinter/linter-jscs) package, v4.0.4

### Stack Trace

Uncaught TypeError: Object.defineProperty called on non-object

```
At /Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:1002

TypeError: Object.defineProperty called on non-object
    at Function.defineProperty (native)
    at Object.exports.getContent (/Users/user/.atom/packages/linter-jscs/node_modules/jscs/lib/cli-config.js:115:16)
    at Object.exports.load (/Users/user/.atom/packages/linter-jscs/node_modules/jscs/lib/cli-config.js:139:20)
    at Function.getConfig (/Users/user/.atom/packages/linter-jscs/src/linter-jscs.js:237:27)
    at /Users/user/.atom/packages/linter-jscs/src/linter-jscs.js:104:31
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:125:28)
    at TextBuffer.module.exports.TextBuffer.saveAs (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1091:20)
    at TextBuffer.module.exports.TextBuffer.save (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1083:19)
    at TextEditor.module.exports.TextEditor.save (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:660:26)
    at Pane.module.exports.Pane.saveItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:718:18)
    at Pane.module.exports.Pane.saveActiveItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:701:19)
    at Workspace.module.exports.Workspace.saveActivePaneItem (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace.js:684:35)
    at atom-workspace.commandRegistry.add.core:save (/Applications/Atom.app/Contents/Resources/app.asar/src/register-default-commands.js:214:32)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:580:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:388:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:98:36)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```

### Commands

```
  5x -0:48.7.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:47.5.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:46.5.0 core:move-up (atom-text-editor.editor.is-focused)
     -0:45.2.0 pane:show-item-2 (atom-text-editor.editor.is-focused)
     -0:39.8.0 core:save (atom-text-editor.editor.is-focused)
     -0:18.4.0 core:move-to-top (atom-text-editor.editor.is-focused)
  2x -0:17.9.0 editor:newline (atom-text-editor.editor.is-focused)
  2x -0:17.4.0 core:move-up (atom-text-editor.editor.is-focused)
     -0:16.5.0 core:paste (atom-text-editor.editor.is-focused)
 28x -0:14.8.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:10.1.0 editor:move-to-first-character-of-line (atom-text-editor.editor.is-focused)
     -0:09.7.0 core:move-down (atom-text-editor.editor.is-focused)
     -0:09.4.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:08.7.0 core:move-up (atom-text-editor.editor.is-focused)
 28x -0:07.6.0 core:backspace (atom-text-editor.editor.is-focused)
  2x -0:03.6.0 core:save (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "local-server-web-view"
    ],
    "themes": [
      "seti-ui",
      "firewatch-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
expose, v0.12.1
file-icons, v1.7.15
firewatch-syntax, v1.0.1
highlight-selected, v0.11.2
javascript-snippets, v1.2.1
linter, v1.11.14
linter-csslint, v1.3.2
linter-erb, v1.0.2
linter-htmlhint, v1.2.1
linter-jscs, v4.0.4
linter-jshint, v2.1.0
linter-ruby, v1.2.2
minimap, v4.24.6
minimap-highlight-selected, v4.4.0
pigments, v0.29.2
react, v0.15.0
seti-ui, v1.3.1
terminal-plus, v0.14.5

# Dev
No dev packages
```

**Comment by Arcanemagus** [comment]
That error is being thrown from JSCS itself. Does it work from the CLI?

**Comment by PowerlessCube** [comment]
Hey there,

I'm relatively new to coding and part of our course is working with React.
Previously, I used linter-JShint to catch most of my typos but I've found
since I downloaded linter jscs it's too strict inregards to what it
considers typos.  I kept getting indentation errors in my jsx files.  It
would display the browser fine with no errors, but my code would be covered
in linter errors.

I ended up uninstalling as I was spending more time making exceptions to
some of the rules than actually coding.

-PC
